includes/libft.h

```diff
@@ -44,6 +44,7 @@ char				*ft_strncat(char *s1, const char *s2, size_t n);
 size_t				ft_strlcat(char *dst, const char *src, size_t size);
 char				*ft_strchr(const char *s, int c);
 char				*ft_strrchr(const char *s, int c);
+size_t				ft_strchri(const char *str, char c);
```

Comment:
What is ft_strchri doing?

Reply:
It counts the amount of times the char c that you send, is present in the string.